src/components/admin/UserProfileModal/components/EditingProfileModalContent.tsx

```diff
@@ -166,49 +165,55 @@ export const EditingProfileModalContent: React.FC<CurrentUserProfileModalContent
-        editMode
-        containerClassName="EditingProfileModalContent__section"
-        questions={questions}
-        answers={answers}
-        register={register}
-      />
-      <ProfileModalEditLinks
-        containerClassName="EditingProfileModalContent__section"
-        register={register}
-        initialLinks={defaultValues.profileLinks ?? []}
-        links={links}
-        errors={errors?.profileLinks}
-        onDeleteLink={onDeleteLink}
-        onAddLink={addLinkHandler}
-      />
-      {isChangePasswordShown && (
-        <ProfileModalChangePassword
+    <div className="EditingProfileModalContent">
+      <ModalTitle>Edit profile</ModalTitle>
+
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <ProfileModalEditBasicInfo
+          user={user}
+          containerClassName="EditingProfileModalContent__section"
+          register={register}
+          watch={watch}
+          setValue={setValue}
+          partyNameError={errors?.partyName}
+        />
+        <ProfileModalQuestions
+          editMode
+          containerClassName="EditingProfileModalContent__section"
+          questions={questions}
+          answers={answers}
+          register={register}
+        />
+        <ProfileModalEditLinks
           containerClassName="EditingProfileModalContent__section"
           register={register}
-          getValues={getValues}
-          errors={pick<
-            FieldErrors<UserProfileModalFormData>,
-            "oldPassword" | "newPassword" | "confirmNewPassword"
-          >(errors, ["oldPassword", "newPassword", "confirmNewPassword"])}
+          initialLinks={defaultValues.profileLinks ?? []}
+          links={links}
+          errors={errors?.profileLinks}
+          onDeleteLink={onDeleteLink}
+          onAddLink={addLinkHandler}
         />
-      )}
-      <EditProfileModalButtons
-        isChangePasswordShown={!isChangePasswordShown}
-        onChangePasswordClick={showChangePassword}
-        onCancelClick={cancelEditing}
-        isSubmitting={isSubmitting}
-        containerClassName="EditingProfileModalContent__edit-buttons"
-      />
-    </form>
+        {isChangePasswordShown && (
+          <ProfileModalChangePassword
+            containerClassName="EditingProfileModalContent__section"
+            register={register}
+            getValues={getValues}
+            errors={pick<
+              FieldErrors<UserProfileModalFormData>,
+              "oldPassword" | "newPassword" | "confirmNewPassword"
+            >(errors, ["oldPassword", "newPassword", "confirmNewPassword"])}
+          />
+        )}
+        <div className="mt-5 sm:mt-4 sm:flex sm:flex-row-reverse">
+          <EditProfileModalButtons
+            isChangePasswordShown={!isChangePasswordShown}
+            onChangePasswordClick={showChangePassword}
+            onCancelClick={cancelEditing}
+            isSubmitting={isSubmitting}
+            containerClassName="EditingProfileModalContent__edit-buttons"
```

Comment:
Can we use `.tailwind.ts` file instead?

Reply:
Or maybe just remove it.

Reply:
hmm, not sure what should we do with these class names... I wouldn't like to remove them, to be honest. They help us navigate through the page when we open inspector in the developer tools, and then we can quickly search for the element in the code. 
I know that there's a way to do the same via React Tools, but it also adds a bit more meaning to our HTML.

@denisdimitrov, @goran-peoski-work, what do you think about these class names? component class names, etc? should we remove them?

(can answer the same to the rest of the unresolved comments)

Reply:
> hmm, not sure what should we do with these class names... I wouldn't like to remove them, to be honest. They help us navigate through the page when we open inspector in the developer tools, and then we can quickly search for the element in the code.

This is also a point Mike and I made, Collin didn't think so much of it. But I do know he was open to using `data-` attributes (if maybe for another case), but we might try doing that instead, offload our own data needs from the style logic.

I have done this in a hobby project (still small, so manageable):

I put  something like `<div data-dn={dn} />` at the top level. It may be any top level tag, and `dn` is just the component's `displayName` and I try to make it the first attribute. For children I've used another one `data-el={}` so that I could have the block and the element inside it (according to BEM).

Reply:
@denisdimitrov, I'd like just to add that @goran-peoski-work and I decided on huddle that we're going to use `data-bem` attribute where we can move these meaningful class names. Feel free to use that approach too. 🙃

Reply:
as agreed, moved class name info to `data-bem`:  https://github.com/sparkletown/sparkle/pull/2931/commits/c90bfc174c869b913ac93ae9776b886e624279c8